lib/features/campaigns/screens/map_consumer.dart

```diff
@@ -34,26 +33,32 @@ typedef GetPoiEditWidgetCallback<T> = Widget Function(T);
 typedef OnDeletePoiCallback = Future<void> Function(String poiId);
 
 abstract class MapConsumer<T extends StatefulWidget, PoiCreateType, PoiDetailType, PoiUpdateType> extends State<T>
-    with InfoBox, SearchMixin<T> {
+    with
+        InfoBox,
+        SearchMixin<T>,
+        MapConsumerActionAreaMixin,
+        MapConsumerExperienceAreaMixin,
+        MapConsumerRouteMixin,
+        MapConsumerFocusAreaMixin,
+        MapConsumerPollingStationMixin {
   late MapController mapController;
 
   final NominatimService _nominatimService = GetIt.I<NominatimService>();
 
-  bool focusAreasVisible = false;
-  bool pollingStationVisible = false;
-  bool routesVisible = false;
-  bool experienceAreasVisible = false;
-
   final _minZoomFocusAreaLayer = 11.0;
   final _minZoomPollingStationLayer = 11.0;
   final _minZoomRouteLayer = 11.0;
+  final _minZoomExperienceAreaLayer = 11.0;
```

Comment:
Do we think there is a future where different map elements have different min zoom levels?